CHANGES-DNF-5.md

```diff
@@ -49,6 +49,14 @@ options and implemented only for related commands
 Renaming boolean options to format `--<option>`, and `--no-<option>`
 `--nobest` -> `--no-best`. Proposing to keep `--nobest` as a deprecated option.
 
+### --strict deprecation
+`--strict` option is now deprecated. The problem with this option is that it does two things:
+
+1. it allows the solver to skip uninstallable packages to resolve depsolv problems
+2. it allowd dnf to skip unavailable packages (this is for `install` command mostly)
+
+The functionality is now splitted to two options - `--skip-broken` for the uninstallable packages and `--skip-unavailable` for packages not present in repositories.
```

Comment:
I think that while `splitted` is a word just `split` would [be better](https://www.usingenglish.com/reference/irregular-verbs/split.html).